source/fundamentals/builders/sort.txt

```diff
@@ -11,3 +11,320 @@ Sorts Builders
+
+Text Score
+----------
+
+If you are using :manual:`$text </reference/operator/query/text/>`
+queries to search your data you can sort the results of those
+queries by text score (how well your search matched the returned document).
+To sort your results by the
+:manual:`$meta </reference/operator/aggregation/meta/#exp._S_meta>`
+``"textScore"`` associated with a ``$text`` query, use the
+``Sorts.metaTextScore()`` static factory method to retrieve a sort specification
+for a descending sort by score (best match first).
+
+.. note::
+
+  If your MongoDB instance is running MongoDB 4.2 and earlier,
+  to use a ``Sorts.metaTextScore()``
+  expression with ``FindIterable.sort()``, you must also
+  include the same expression in a projection. This is demonstrated
+  in the code example below.
+
+The ``$text`` command performs a text search on the content indexed by the 
+:manual:`text index </core/index-text/>` of your collection. You must create
+a text index on your collection before you can run a ``$text`` query and use
+``Sorts.metaTextScore()``.
+
+.. example::
+
+   In the following code example, we show how to use the
+   ``metaTextScore()`` method to sort the results of a ``$text``
+   query on ``sort_example``. The following code snippet
+   works as follows:
+
+   #. Create a text index for ``sort_example`` on the ``"food"`` field.
+   #. Run a ``$text`` search for the phrase ``"maple donut"``.
+   #. Project ``metaTextScore("score")`` into your query results.
+      This step is optional if your MongoDB instance is running MongoDB 4.4 or later. 
+   #. Sort using ``metaTextScore("score")`` as a sort specification.
+
+   .. code-block:: java
+   
+      import static com.mongodb.client.model.Sorts.metaTextScore;
+      import static com.mongodb.client.model.Sorts.ascending;
+   
+      // <MongoCollection setup code here>
+      
+      // create a text index on the food field
+      collection.createIndex(Indexes.text("food"));
+      // create text score sort specification
+      Bson metaTextScore = metaTextScore("score");
+      String searchTerm = "maple donut";
+      // build $text search query
+      Bson searchQuery = new Document("$text", new Document("$search", searchTerm));
+      collection.find(searchQuery)
+              // project the text score into documents on the "score" field (optional in MongoDB 4.4 or later)
+              .projection(metaTextScore)
+              // sort the documents on the "textScore" field
+              .sort(metaTextScore)
+              .into(results);
+      for (Document result : results) {
+          System.out.println(result.toJson());
+      }
+   
+   The output of the above code snippet should look something like this: 
+   
+   .. code-block:: none
+
+      {"_id": 6, "letter": "c", "food": "maple donut", "score": 1.5}
+      {"_id": 2, "letter": "a", "food": "donuts and coffee", "score": 0.75}
+      {"_id": 3, "letter": "a", "food": "maple syrup", "score": 0.75}
```

Comment:
I think it could be helpful to provide a short summary (as short as one sentence) of the section and/or the entire document.

Reply:
Thanks for the note. Looking at other articles, it seems like the convention is for closing sections to take the form "For more information, see <link to relevant MongoDB content>". I added a similar section here:  https://github.com/mongodb/docs-java/pull/59/commits/cec9d340d6240aeadce6e9d97bed0d12ce42d63b